app/app.go

```diff
@@ -1609,8 +1609,10 @@ func (app *App) getFinalizeBlockResponse(appHash []byte, events []abci.Event, tx
 		}),
 		ConsensusParamUpdates: &tmproto.ConsensusParams{
 			Block: &tmproto.BlockParams{
-				MaxBytes: endBlockResp.ConsensusParamUpdates.Block.MaxBytes,
-				MaxGas:   endBlockResp.ConsensusParamUpdates.Block.MaxGas,
+				MaxBytes:                       endBlockResp.ConsensusParamUpdates.Block.MaxBytes,
+				MaxGas:                         endBlockResp.ConsensusParamUpdates.Block.MaxGas,
+				CosmosGasMultiplierNumerator:   endBlockResp.ConsensusParamUpdates.Block.CosmosGasMultiplierNumerator,
```

Comment:
do we need migration for new consensus params?

Reply:
@codchen we removed this in favor of just adding in a new param in cosmos layer: https://github.com/sei-protocol/sei-cosmos/pull/487